src/components/action-group/action-group.tsx

```diff
@@ -50,6 +60,8 @@ export class ActionGroup implements ConditionalSlotComponent {
 
   /**
    * Specifies a text string for the `calcite-action-menu`.
+   *
+   * @deprecated - translations are now built-in, if you need to override a string, please use `messageOverrides`
```

Comment:
nitpick: For the `@deprecated` translation message, can we add a capital on the first letter and a period at the end for API style consistency? Added a recommended refactored line below for use across components, if its possible to integrate easily:

> @deprecated - Translations are built-in. To override the built-in message string, use `messageOverrides`.

Reply:
These will be removed in #5946. Can we skip this for now?

Reply:
Yes, definitely OK to skip. Thanks for clarifying on the removal PR 👍🏻